models/form.js

```diff
@@ -4,14 +4,14 @@ var Schema = mongoose.Schema;
 var formSchema = Schema({
   _studentID: { type: Schema.Types.ObjectId, ref: 'Student' },
   period: String,
-  date: Date,
+  date: Number,
```

Comment:
should we do this conversion client side then? i feel like we still probably want to store it in the DB as a date